### PR TITLE
integration tests to check min-bq argument is hooked up to HC and M2

### LINKED ISSUE #4136: Add HC/Mutect2 integration tests for --min-base-quality-score

### PR DESCRIPTION
**Pull request by davidbenjamin**
Closes #4136.

@droazen Here you go.

### CODE REVIEW

File: src/test/java/org/broadinstitute/hellbender/tools/walkers/haplotypecaller/HaplotypeCallerIntegrationTest.java [hunk 1]
```diff
@@ -499,6 +502,49 @@ public void testAssemblyRegionAndActivityProfileOutput() throws Exception {
...
+                "--min-base-quality-score", "30"
+        };
+
+        runCommandLine(highThresholdArgs);
+
+        final List<VariantContext> variantsWithLowThreshold =
+                StreamSupport.stream(new FeatureDataSource<VariantContext>(outputAtLowThreshold).spliterator(), false).collect(Collectors.toList());
+
+        final List<VariantContext> variantsWithHighThreshold =
+                StreamSupport.stream(new FeatureDataSource<VariantContext>(outputAtHighThreshold).spliterator(), false).collect(Collectors.toList());
```
Comment (droazen) [inline src/test/java/org/broadinstitute/hellbender/tools/walkers/haplotypecaller/HaplotypeCallerIntegrationTest.java]: Close data sources when done using a try-with-resources block.

File: src/test/java/org/broadinstitute/hellbender/tools/walkers/mutect/Mutect2IntegrationTest.java [hunk 2]
```diff
@@ -243,8 +243,56 @@ public void testBamWithRepeatedReads() {
...
+                "--min-base-quality-score", "30"
+        };
+
+        runCommandLine(highThresholdArgs);
+
+        final List<VariantContext> variantsWithLowThreshold =
+                StreamSupport.stream(new FeatureDataSource<VariantContext>(outputAtLowThreshold).spliterator(), false).collect(Collectors.toList());
+
+        final List<VariantContext> variantsWithHighThreshold =
+                StreamSupport.stream(new FeatureDataSource<VariantContext>(outputAtHighThreshold).spliterator(), false).collect(Collectors.toList());
```
Comment (droazen) [inline src/test/java/org/broadinstitute/hellbender/tools/walkers/mutect/Mutect2IntegrationTest.java]: Close data sources when done using a try-with-resources block.

**Comment by droazen** [comment]
Did you confirm that these tests actually fail without your bug fix to hook up the argument, @davidbenjamin ?

**Comment by davidbenjamin** [comment]
> Did you confirm that these tests actually fail without your bug fix to hook up the argument, @davidbenjamin ?

Yes, they fail.

**Pull request merged**